secant/Features/BalanceBreakdown/BalanceBreakdownStore.swift

```diff
@@ -9,65 +9,38 @@ import Foundation
 import ComposableArchitecture
 import ZcashLightClientKit
 
-typealias BalanceBreakdownReducer = Reducer<BalanceBreakdownState, BalanceBreakdownAction, BalanceBreakdownEnvironment>
-typealias BalanceBreakdownStore = Store<BalanceBreakdownState, BalanceBreakdownAction>
-typealias BalanceBreakdownViewStore = ViewStore<BalanceBreakdownState, BalanceBreakdownAction>
+typealias BalanceBreakdownStore = Store<BalanceBreakdown.State, BalanceBreakdown.Action>
 
-// MARK: - State
-
-struct BalanceBreakdownState: Equatable {
-    var autoShieldingTreshold: Zatoshi
-    var latestBlock: String
-    var shieldedBalance: WalletBalance
-    var transparentBalance: WalletBalance
+struct BalanceBreakdown: ReducerProtocol {
+    private enum CancelId {}
     
-    var totalBalance: Zatoshi {
-        shieldedBalance.total + transparentBalance.total
+    struct State: Equatable {
+        var autoShieldingTreshold: Zatoshi
+        var latestBlock: String
+        var shieldedBalance: WalletBalance
+        var transparentBalance: WalletBalance
+        
+        var totalBalance: Zatoshi {
+            shieldedBalance.total + transparentBalance.total
+        }
     }
-}
-
-// MARK: - Action
 
-enum BalanceBreakdownAction: Equatable {
-    case onAppear
-    case onDisappear
-    case synchronizerStateChanged(WrappedSDKSynchronizerState)
-    case updateLatestBlock
-    case updateSynchronizerStatus
-}
-
-// MARK: - Environment
-
-struct BalanceBreakdownEnvironment {
-    let numberFormatter: WrappedNumberFormatter
-    let SDKSynchronizer: WrappedSDKSynchronizer
-    let scheduler: AnySchedulerOf<DispatchQueue>
-}
+    @Dependency(\.numberFormatter) var numberFormatter
+    @Dependency(\.SDKSynchronizer) var SDKSynchronizer
```

Comment:
I am not sure if `SDKSynchronizer` should start with capital letter. It looks like type and not variable then. I suggest to rename this to simply `synchronizer`.

Reply:
As discussed, sdkSynchronizer it is.